packages/callout-block/src/CalloutBlock.tsx

```diff
@@ -117,6 +117,19 @@ export const CalloutBlock = ({ appBridge }: BlockProps): ReactElement => {
                     plugins={getDefaultPluginsWithLinkChooser(appBridge)}
                 />
             </div>
+
+            {/* Hacky way to remove margin from the last element, so that the text is vertically centered in the callout block */}
```

Comment:
Shouldn't it also remove margin-top from the first-child?

Reply:
Good point, added 👍🏻

Reply:
I don't see any classnames on first-child. Did you forget to commit it? Or I'm I missing sth?
![image](https://github.com/Frontify/guideline-blocks/assets/42832501/7f0d0f7c-722f-4240-ba9b-02a05408886a)

Reply:
👍